### PR TITLE
Remove the feedback/accountability partner category

### PR DESCRIPTION
**Pull request by lpatmo**
See: #29

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@babel/runtime": "7.2.0",
     "@material-ui/core": "^3.9.1",
+    "@material-ui/icons": "^3.0.2",
```
Comment (angelocordon) [inline package.json:17]: Considering the main goal of this PR, is this worth committing here?

Comment (lpatmo) [reply package.json:17]: Hmm, I'm not sure why the material-ui version updated.

**Comment by angelocordon** [comment]
@lpatmo - might be worth just tackling #29 in your current PR #42?

**Comment by lpatmo** [comment]
Sounds good! I'll take care of it in #42.

**Pull request closed without merging**